core/src/main/java/com/adobe/aio/auth/JwtContext.java

```diff
@@ -13,37 +13,28 @@
 
 import java.security.PrivateKey;
 import java.util.HashSet;
-import java.util.Map;
 import java.util.Objects;
-import java.util.Properties;
 import java.util.Set;
 
 import org.apache.commons.lang3.StringUtils;
-import org.apache.commons.lang3.builder.EqualsBuilder;
-import org.apache.commons.lang3.builder.HashCodeBuilder;
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 
-import static com.adobe.aio.util.FileUtil.*;
-
 /**
  * JWT Authentication context.
  */
 public class JwtContext implements Context {
-  public static final String CREDENTIAL_ID = "aio_credential_id";
-  public static final String CLIENT_SECRET = "aio_client_secret";
+
   public static final String TECHNICAL_ACCOUNT_ID = "aio_technical_account_id";
   public static final String META_SCOPES = "aio_meta_scopes";
 
-  private final String credentialId;
   private final String technicalAccountId;
   private final Set<String> metascopes;
   private final String clientSecret;
   private final PrivateKey privateKey;
 
-  public JwtContext(final String credentialId, final String clientSecret, final String technicalAccountId,
```

Comment:
im confused here. is credential id = client id? and why is that not needed anymore?

Reply:
the `credentiald` 
* is not needed to build a JwtContext
* was moved as an optional field of the `Worskspace` class
  * as it might be used for other use case (it is used for instance in the events registration v1 api)